doc/contribute.rst

```diff
@@ -37,6 +37,12 @@ project. Here's where you'll find how the development team works and the
 guidelines they (and you) use to contribute code (and documentation) to
 the project.
 
+The project has a Technical Steering Committee (TSC) that is responsible
```

Comment:
This change should go into the ``developer-guides/contribute_guidelines.html`` document, not here.

Reply:
This is not really a guideline, it's a description of how the body that governs the technical direction of the project. I think it would be kind of lost in that document which talks about commit messages, etc.